axon-server-connector/src/main/java/org/axonframework/axonserver/connector/util/AxonFrameworkVersionResolver.java

```diff
@@ -0,0 +1,86 @@
+package org.axonframework.axonserver.connector.util;
+
+
+import org.axonframework.util.MavenArtifactVersionResolver;
+
+import java.io.IOException;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.function.Function;
+import java.util.function.Supplier;
+
+/**
+ * Provides Axon Framework current version.
+ * <p>
+ * At first it looks for the version in the Meta Data of the maven artifact.
+ * As a fallback it looks for the version in the environment variables.
+ *
+ * @author Sara Pellegrini
+ * @since 4.2.1
+ */
+public class AxonFrameworkVersionResolver implements Supplier<String> {
+
+    private static final String ORG_AXONFRAMEWORK = "org.axonframework";
```

Comment:
Nit: I like the constant, but now it's just a rename of what it is exactly. I feel that `AXON_FRAMEWORK_GROUP_ID` would fit better here.